hermes/probe/delete_file.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Delete_file implements the probe operation for deleting a file in a
+// storage system.
+// Package probe implements the probe that Hermes uses to monitor
```

Comment:
"// Package probe" must be the first line in the file.
Also note that this package comment only has to be present in one of the files belonging to the package.
The typical pattern is to have a single file per package with the file named exactly like the package.

Reply:
I've moved this into a `delete` package and adjusted the package comment appropriately.